Project21/Project21/Assignment 3.c

```diff
@@ -0,0 +1,54 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment 3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// Assignment 3
+// Decimal to Binary conversion and Decimal to HexaDecimal conversion
+// ------------------------------------------------------------------------------------------------
+#include<stdio.h>
+#define INT_SIZE sizeof(int)*8
+#define HEX_SIZE 8
+
+/// <summary>Function to convert a decimal number to binary</summary>
+int DecimalToBinary (int n) {
+   int index = 0;
+   int binary[INT_SIZE];
+   int num = n;
+   index = INT_SIZE - 1;
+   while (index >= 0) {
+      binary[index--] = n & 1;// Extract the least significant bit (LSB) from 'n' 
+      n >>= 1;// Right-shift 'n' by one bit to remove the extracted bit 
+   }
+   printf ("Binary value: ");

```

Comment:
Inside the functions you should not print the results. Return the result to main and print it there.